tests/library/Garp/Validate/GreaterThanOrEqualToTest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Garp_Validate_GreaterThanOrEqualToTest
+ * Tests Garp_Validate_GreaterThanOrEqualTo
+ *
+ * @package Tests
+ * @author  Han Kortekaas <han@grrr.nl>
+ * @group   Validate
+ */
+class Garp_Validate_GreaterThanOrEqualToTest extends Garp_Test_PHPUnit_TestCase {
+
+    /**
+     * @dataProvider numberProvider
+     *
+     * @param int|float $value
+     * @param int|float $min
+     * @param string
+     */
+    public function testIsValid($value, $min, $assertion) {
+        $validator = new Garp_Validate_GreaterThanOrEqualTo($min);
+
+        $this->$assertion($validator->isValid($value));
+    }
+
+    public function numberProvider() {
+        return [
+            [2,   1,   'assertTrue'],
```

Comment:
I would have used `true`/`false` and `$this->assertSame`, but this makes the data provider more readable. I like it.

Reply:
Also my first instinct, but that somehow seemed a bit of an anti-pattern when there are these dedicated functions. Glad you like it.